### PR TITLE
(Not for submission) Interceptor demo

### PR DESCRIPTION
**Pull request by jskeet**
This is based on #2201 - only the last commit is new.

### CODE REVIEW

**Comment by jskeet** [comment]
(Ignore the solution file change, too :)

File: apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1/LanguageServiceClient.cs [hunk 1]
```diff
@@ -58,6 +58,11 @@ private LanguageServiceSettings(LanguageServiceSettings existing) : base(existin
             OnCopy(existing);
         }
 
+        /// <summary>
+        /// This will be in GAX...
+        /// </summary>
+        public grpccore::Interceptors.Interceptor Interceptor { get; set; }
```
Comment (AnashOommen) [inline apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1/LanguageServiceClient.cs]: If the setter is public here, does it mean that a user can change the interceptor after using Create(channel, settings);

Comment (jskeet) [reply apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1/LanguageServiceClient.cs]: Yes, but it won't have any impact. That's true of most (all?) service-wide settings.

**Comment by chrisdunelm** [comment]
LGTM, this is exactly what I was expecting :)

**Comment by jskeet** [comment]
Closing now this has been implemented (but not yet merged).

**Pull request closed without merging**